src/jgikbase/idmapping/storage/id_mapping_storage.py

```diff
@@ -11,6 +11,8 @@
 from typing import List
 from jgikbase.idmapping.core.namespaced_id import NamespacedID
 
+# TODO make the coverage calculation not care about abstract classes
```

Comment:
I think if you add `#  pragma: no cover` after the class name, it will ignore the class

Reply:
I'm just going to cover this with tests in a later PR. I was under the impression that you could instantiate an abstract class, but in fact you can.

Reply:
It turns out mypy won't let you instantiate an abstract class, but python will, so I'm going with your approach.